src/lib/modules/Document/DocumentCard/DocumentCard.js

```diff
@@ -39,6 +39,15 @@ class DocumentCard extends Component {
     const { data } = this.props;
     const { metadata } = data;
     const url = FrontSiteRoutes.documentDetailsFor(metadata.pid);
+    let authors = data.metadata.authors.map((elem) => {
```

Comment:
In case we have a very high number of authors, better to slice first:
```
const authors = data.metadata.authors
    .slice(0, invenioConfig.LITERATURE.authors.maxDisplay)
    .map((elem) => elem.full_name)
    .join('; ');
```